examples/interpretation/notebooks/Dutch_F3_patch_model_training_and_evaluation.ipynb

```diff
@@ -117,17 +117,7 @@
    "outputs": [],
    "source": [
     "import numpy as np\n",
```

Comment:
Looks like you need to merge earlier commits

Reply:
looks good to me... what's the problem?

Reply:
I think Sharat meant that these changes were already made in PR #247.